poggler.rtlr

```diff
@@ -1,27 +1,7 @@
 require 'rattler'
-require_relative './parser_parts/variable_binding'
```

Comment:
this was getting daunting and tweaking these all as I moved parts around was not exciting.